fabric-renderer-indigo/src/main/java/net/fabricmc/fabric/impl/client/indigo/renderer/mesh/MutableQuadViewImpl.java

```diff
@@ -53,7 +53,8 @@ public final void begin(int[] data, int baseIndex) {
 		clear();
 	}
 
-	public void clear() {
```

Comment:
Breaking change

Reply:
Wait this is impl lmao

Reply:
This is an impl class though, so we don't care?